File: tests/bake-gettext/python-simple/source.py
import gettext

x1 = gettext.gettext("gettext")
x2 = gettext.lgettext("lgettext")
m1 = gettext.gettext("""multi
line
string""")
Z = 'Z'
x5 = gettext.gettext(Z) # Ignored, must be string constants

File: tests/stew-gettext/c-simple/source.c
#include <libintl.h>

#define Z "Z"

int main (int argc, char **argv)
{
    char *x1 = gettext("gettext");
    char *x4 = gettext(Z); /* Ignored, must be string constants */
    char *x5 = "Z"; /* Ignored, not in a gettext function */
    char *x6 = "gettext("Z")"; /* Ignored, inside a string constant */

    return 0;
}
